packages/components/field-label/src/field-label.tsx

```diff
@@ -119,7 +132,10 @@ const FieldLabel = (props: TFieldLabelProps) => {
         </Inline>
 
         {props.hint && (
-          <Inline alignItems="flexStart" scale="xs">
+          <Inline
+            alignItems={theme === 'default' ? 'flexStart' : 'center'}
+            scale="xs"
+          >
             {props.hintIcon && (
               <Inline>
                 {cloneElement(props.hintIcon, {
```

Comment:
I think we're missing here using the square shape for the icon in the new theme.

Reply:
Good point. I assumed we will continue with the rounded buttons first, as it was not that easy to change to squared ones everywhere. I would rather prefer to keep the rounded ones, than having a mix of squared and rounded